packages/turf-area/index.ts

```diff
@@ -1,9 +1,11 @@
-import { Feature, FeatureCollection, Geometry } from "@turf/helpers";
+import {
+  Feature,
+  FeatureCollection,
+  Geometry,
+  earthRadius,
+} from "@turf/helpers";
 import { geomReduce } from "@turf/meta";
 
-// Note: change RADIUS => earthRadius
-const RADIUS = 6378137;
```

Comment:
Thanks for your reply and suggestions.
I have just tried the commands `REGEN=true yarn test` and `yarn prettier --write ./test/out/` in the package turf-nearest-neighbor-analysis and the result is the same of my previous commit, and all 6 tests pass.
Previously, I've done exactly the same but I used `npm` instead of yarn, with equal results.

I've inspected the section related to the tests of turf-nearest-neighbor-analysis in the CI console output log, where the errors are located, and it sees that it fails because the output used in the asserts is still the old one (and not the one I've committed in the directory /test/out). 
So, why is the CI not looking to the latest changes I made on this branch?

Reply:
Either you need to run `yarn build` in the turf-area package, or you need to commit the updated test output files in the other package.

Reply:
I've committed the updated test output files in commits https://github.com/Turfjs/turf/pull/2047/commits/4bd33e6e99e03614eac4c6e10d95c9a21a8c5053 and https://github.com/Turfjs/turf/pull/2047/commits/88a0a03a63729fa265f4933971044ab26688a411
Maybe I'm missing any step?

Reply:
It looks like you regenerated them and committed them without prettier and then ran prettier and committed that, but if you look in this PR's list of files changed they aren't listed. That makes me think you might've run the test and prettier commands in that package before running build in turf-area.

Reply:
I've looked into the commits on the turf-nearest-neighbor-analysis package and there is this message:
"This commit does not belong to any branch on this repository" so this is why their changes are not being picked up by the CI. I've tried a few git commands to add the files of that commit to the branch but without success.
Any ideas on how to achieve that?